conf/distro/include/rpb.inc

```diff
@@ -7,6 +7,12 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 PACKAGECONFIG_append_pn-systemd = " resolved networkd"
 PACKAGECONFIG_remove_pn-gpsd = "qt"
 
+# FIXME Mali is currently added as a distro config on a per-machine basis
```

Comment:
hmm. why do we have that in the distro config, in the first place? if we omit that fact, the change is ok for me..